data-processing/pdf/grobid_client.py

```diff
@@ -0,0 +1,24 @@
+import requests
+import json
+
+def get_pdf_structure(pdf_file):
+    """
+    Get parsed PDF data from a local instance of the Grobid service
+    :param pdf_file:
+    :return:
+    """
+    try:
+        files = {'input':(pdf_file, open(pdf_file,'rb'), 'application/pdf',{'Expires':'0'})}
+        resp = requests.post('http://localhost:8070/api/processPdfStructure',files=files)
+
+        if resp.status_code == 200:
+            return json.loads(resp.text)
+        else:
+            raise Exception("Grobid returned status code {}".format(resp.status_code))
+    except requests.exceptions.ConnectionError:
+        msg="""Could not connect to local grobid service.
```

Comment:
Running the PDF pipeline only requires pulling and running this Grobid Docker container.

Reply:
Sweet! Loving the embedded docs 👍